apollo-mpp-utils/src/appleMain/kotlin/com/apollographql/apollo3/mpp/utils.kt

```diff
@@ -3,14 +3,15 @@ package com.apollographql.apollo3.mpp
 import platform.Foundation.NSDate
 import platform.Foundation.NSDateFormatter
 import platform.Foundation.NSThread
+import platform.Foundation.timeIntervalSince1970
 import platform.posix.pthread_self
 import kotlin.native.concurrent.ensureNeverFrozen
 import kotlin.native.concurrent.freeze
 import kotlin.native.concurrent.isFrozen
 import kotlin.system.getTimeMillis
 
 actual fun currentTimeMillis(): Long {
-  return getTimeMillis()
+  return (NSDate().timeIntervalSince1970 * 1000).toLong()
```

Comment:
[getTimeMillis](https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.system/get-time-millis.html) gets current system time in milliseconds since certain moment in the past. I want the EPOCH here. I'll update the KDoc too while I'm at it.

Reply:
woops my bad!